services/registry/src/generated/gql_configuration_pb.d.ts

```diff
@@ -29,6 +29,16 @@ export class GqlConfigurationMessage extends jspb.Message {
     getSchema(): GqlSchema | undefined;
     setSchema(value?: GqlSchema): void;
 
+    clearEndpointsList(): void;
+    getEndpointsList(): Array<GqlEndpoint>;
+    setEndpointsList(value: Array<GqlEndpoint>): void;
+    addEndpoints(value?: GqlEndpoint, index?: number): GqlEndpoint;
+
+    clearAuthprovidersList(): void;
+    getAuthprovidersList(): Array<GqlAuthProvider>;
+    setAuthprovidersList(value: Array<GqlAuthProvider>): void;
```

Comment:
Not sure why it was generated as `setAuthproviderList` and not `setAuthProviderList`

Reply:
It should be snake case in protobuf file